src/main/java/com/github/stefvanschie/inventoryframework/GuiItem.java

```diff
@@ -139,6 +148,26 @@ public void callAction(@NotNull InventoryClickEvent event) {
     public void setAction(@NotNull Consumer<InventoryClickEvent> action) {
         this.action = action;
     }
+    
+    /**
+     * Returns the list of properties
+     *
+     * @return the list of properties that belong to this gui item
+     */
+    @NotNull
+    @Contract(pure = true)
+    public List<Object> getProperties(){
+        return properties;
+    }
+    
+    /**
+     * Sets the list of properties for this gui item
+     *
+     * @param list of new properties
```

Comment:
This param tag refers to the `list` parameter, but the parameter is named `properties`. This should be changed so it refers to the parameter by the right name, so javadocs generation doesn't fail.

Reply:
yeah i never really used javadoc myself, so that slipped past me, should be good now